packages/react-vapor/src/components/tooltip/examples/TooltipExamples.1.use.md

```diff
@@ -0,0 +1,19 @@
+## Guidelines
+
+### Do
+
+-   Keep the **text short** and direct (3 lines maximum). For more detailed explanations, link to external documentation.
+-   Make Tooltips **appear in empty areas** in the screen, or above less relevant information.
```

Comment:
```suggestion
-   Make Tooltips **appear in empty areas** of the screen, or above less relevant information.
```

Reply:
Suggested change: 
" Have Tooltips **appear over empty areas** or over less relevant information on the page. "